src/Root.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+
+export const Root = () => (
+  <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="home" element={<Navigate to=".." />} />
+        <Route path="people">
+          <Route index element={<PeoplePage />} />
+          <Route path=":slug?" element={<PeoplePage />} />
+        </Route>
+        <Route path="*" element={<NotFoundPage />} />
+      </Route>
+    </Routes>
+    <App />
```

Comment:
The `App` component is rendered twice: once as a route element and once directly inside the `Router`. This redundancy might lead to unexpected behavior or duplicate rendering. Consider removing the direct rendering of `<App />` on this line.